scripts/fontawesome5.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+echo "Please enter your FontAwesome5 npm token:"
+
+read fa5_token
+
+echo "Setting up npm config"
+
+npm config set "@fortawesome:registry" https://npm.fontawesome.com/
+npm config set "//npm.fontawesome.com/:_authToken" $fa5_token
+
+echo "Installing FontAwesome5"
+
+npm install @fortawesome/fontawesome-free --no-shrinkwrap
```

Comment:
Think we can do `npm pack` instead to avoid polluting the `node_modules` folder

Reply:
```npm pack``` is probably better yea...